AndroidAnnotations/functional-test-1-5/src/test/java/org/androidannotations/test15/SeekBarChangeListenedActivityTest.java

```diff
@@ -38,13 +38,16 @@
 	public void setUp() {
 		activity = setupActivity(SeekBarChangeListenedActivity_.class);
 		seekBar = (SeekBar) activity.findViewById(R.id.seekBar1);
-		shadowSeekBar = shadowOf_(seekBar);
+		shadowSeekBar = (ShadowSeekBar) extract(seekBar);
```

Comment:
but you repeat this call with a local member in each test method but `testSeekBarTouchStopNamingConvention` there you use the field yoou aquire in `setUp`. thats what i meant.